Styles/general.css

```diff
@@ -2,35 +2,32 @@
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Righteous&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@300;400;500;600&family=Montserrat:wght@400;500;600&family=Poppins&family=Raleway:wght@600&display=swap');
 
-h1{
+h1 {
     font-size: 2rem;
     font-weight: 700;
     font-family: 'Montserrat Alternates', sans-serif;
 }
 
-h2{
+h2 {
     font-size: 1.5rem;
-    font-weight:500;
+    font-weight: 500;
     font-family: 'Montserrat', sans-serif;
 }
 
-h3{
+h3 {
     font-size: 1rem;
     font-weight: 500;
 }
 
-p{
+p {
     font-size: 1rem;
     font-weight: 500;
     font-family: 'Montserrat', sans-serif;
 }
 
-body{
-    /* font-family: serif; */
+body {
     font-family: 'Comfortaa', cursive;
```

Comment:
Here you are defining font as cursive and later in main.css you are changing it. This will cause import order problems. Delete from here since i see font from photo that you are using the one from main.css

Reply:
ah, thanks for pointing it out, will fix it.